src/components/movies/MovieCard.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+    <div class="movie__card" v-if="movie">
+        <h1 class="movie__title"> {{ movieTitle }}</h1>
```

Comment:
For SEO and accessibility reasons, it is a good practice to have only one `<h1>` tag on a page but I guess there may be multiple of those 😉